dydx-testnet-4/genesis.json

```diff
@@ -0,0 +1,6642 @@
+        {
+          "exponent": -8,
+          "id": 20,
+          "price": 5717635307
+        },
+        {
+          "exponent": -9,
+          "id": 21,
+          "price": 1943019371
+        },
+        {
+          "exponent": -9,
+          "id": 22,
+          "price": 1842365656
+        },
+        {
+          "exponent": -9,
+          "id": 23,
+          "price": 6787621897
+        },
+        {
+          "exponent": -9,
+          "id": 24,
+          "price": 1127629325
+        },
+        {
+          "exponent": -10,
+          "id": 25,
+          "price": 2779565892
+        },
+        {
+          "exponent": -9,
+          "id": 26,
+          "price": 1855061997
+        },
+        {
+          "exponent": -9,
+          "id": 27,
+          "price": 1562218603
+        },
+        {
+          "exponent": -16,
+          "id": 28,
+          "price": 2481900353
+        },
+        {
+          "exponent": -10,
+          "id": 29,
+          "price": 1686998025
+        },
+        {
+          "exponent": -15,
+          "id": 30,
+          "price": 8895882688
+        },
+        {
+          "exponent": -10,
+          "id": 31,
+          "price": 5896318772
+        },
+        {
+          "exponent": -10,
+          "id": 32,
+          "price": 6327613800
+        },
+        {
+          "exponent": -9,
+          "id": 1000000,
+          "price": 1000000000
+        },
+        {
+          "exponent": -9,
+          "id": 1000001,
+          "price": 2050000000
+        }
+      ]
+    },
+    "rewards": {
+      "params": {
+        "denom": "adv4tnt",
+        "denom_exponent": -18,
+        "fee_multiplier_ppm": 990000,
+        "market_id": 11,
+        "treasury_account": "rewards_treasury"
+      }
+    },
+    "sending": {},
+    "slashing": {
+      "missed_blocks": [],
+      "params": {
+        "downtime_jail_duration": "60s",
+        "min_signed_per_window": "0.2",
+        "signed_blocks_window": "12288",
+        "slash_fraction_double_sign": "0.0",
+        "slash_fraction_downtime": "0.0"
+      },
+      "signing_infos": []
+    },
+    "staking": {
+      "delegations": [],
+      "exported": false,
+      "last_total_power": "0",
+      "last_validator_powers": [],
+      "params": {
+        "bond_denom": "adv4tnt",
+        "historical_entries": 10000,
+        "max_entries": 7,
+        "max_validators": 100,
+        "min_commission_rate": "0.05",
+        "unbonding_time": "1036800s"
+      },
+      "redelegations": [],
+      "unbonding_delegations": [],
+      "validators": []
+    },
+    "stats": {
+      "params": {
+        "window_duration": "2592000s"
+      }
+    },
+    "subaccounts": {
+      "subaccounts": []
+    },
+    "transfer": {
+      "denom_traces": [],
+      "params": {
+        "receive_enabled": true,
+        "send_enabled": true
+      },
+      "port_id": "transfer",
+      "total_escrowed": []
+    },
+    "upgrade": {},
+    "vest": {
```

Comment:
do the treasury accounts have some initial balance so that the vesting + rewards work correctly?

Reply:
No funds in treasury, only [vester](https://github.com/dydxprotocol/v4-chain/blob/77573aaceca5bf83d9372b42b64ccf8f11e24dbf/protocol/app/module_accounts_test.go#L32C50-L32C93)

Reply:
It has `10000000000000000000000000` balance or 10million

Reply:
should the [community vester account](https://github.com/dydxprotocol/v4-chain/blob/77573aaceca5bf83d9372b42b64ccf8f11e24dbf/protocol/app/module_accounts_test.go#L34C19-L34C45) have some initial balance too?

Reply:
update: sync offline, 
* no need for the community vester account to have init balance
* the rewards vester balance lgtm